packages/vue/src/components/organisms/SfTopBar/SfTopBar.scss

```diff
@@ -0,0 +1,27 @@
+@import '../../../css/variables';
+
+$top-bar-background-color: $c-light-primary;
+$top-bar-horizontal-padding: $spacer-extra-big;
+
+.sf-topbar {
+  width: 100%;
+  height: 40px;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  box-sizing: border-box;
+  padding: 0px $top-bar-horizontal-padding;
+  background-color: $top-bar-background-color;
+
+  &__center {
+    font-size: $font-size-small-desktop;
+  }
+}
+
+@media only screen and (max-width: $desktop-min) {
```

Comment:
i think you should also handle max-width here with a custom var, with `$desktop-min` as default, wdyt?
```scss
$top-bar-hidden-max-width: $desktop-min !default;
```